Resources/Prototypes/Entities/Objects/Specific/syndicate.yml

```
@@ -148,10 +148,10 @@
   components:
   - type: Store
     balance:
-      Telecrystal: 60
+      Telecrystal: 80 #Sunrise edit
   - type: Tag
     tags:
-    - NukeOpsUplink
+    - LoneOpsUplink #Sunrise added
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)